crox/src/main.rs

```diff
@@ -42,6 +43,8 @@ struct Event {
 #[derive(StructOpt, Debug)]
 struct Opt {
     file_prefix: PathBuf,
+    #[structopt(long = "collapse_threads")]
```

Comment:
Is this necessary?

Reply:
what is not necessary? the command or the PR?

Reply:
o now I understand the attribute was not needed